packages/auth/src/__tests__/auth.test.ts

```diff
@@ -1,6 +1,91 @@
-import Auth from '../index'
+import {rest} from 'msw'
+import {setupServer} from 'msw/node'
+import cookie from '@skillrecordings/cookies'
+import {ACCESS_TOKEN_KEY, USER_KEY} from '@skillrecordings/config'
 
-test('can create an instance', () => {
+import Auth, {getAuthorizationHeader, getLocalUser} from '../index'
+
+export const cornDogUser = {
+  name: 'Corn Dog',
+}
+
+export const server = setupServer(
+  // we can fake the actual user tat gets loaded with msw 🎉
+  rest.get(`/api/users/current`, (req, res, ctx) => {
```

Comment:
[msw](https://mswjs.io/docs/) is rad and makes mocking end points a breeze

amaze!